<!DOCTYPE html>
<html lang="en">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Bootstrap core CSS -->
  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom fonts for this template -->
  <script src="https://kit.fontawesome.com/e51c75a363.js" crossorigin="anonymous"></script>
  <!--<link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css"> -->
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
  <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
  
  <!-- Custom styles for this template -->
  <link href="css/agency.css" rel="stylesheet">

  <style>
  table {
    font-family: arial, sans-serif;
    border-collapse: collapse;
    width: 45%;
    margin: 10px;
  }
  
  td, th {
    border: 1px solid #dddddd;
    text-align: left;
    padding: 8px;
  }
  
  tr:nth-child(even) {
    background-color: #dddddd;
  </style>

  
</head>


  <div class="topheader">
    <div class="row">
      <div class="col-sm lab">
        <!-- Carnevale Lab -->
      </div>
    </div>
  </div>
<body id="page-top">

 <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
    <div class="navbar-logo">
      <a href="https://www.temple.edu"><img src="img/logos/custom_temple_logo.png" class="img-fluid"></a>
    </div>
    <div class="container">
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
        Menu
        <i class="fas fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbarResponsive">
        <ul class="navbar-nav text-uppercase ml-auto">
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="index.html">Home</a>
          </li>
          <li class="nav-item dropdown"">
            <a class="nav-link dropdown-toggle" href="#" " id="navbarDropdown" role="button" data-toggle="dropdown" haspopup="true" expanded="false">
              Research</a>
            <div class="dropdown-menu" labelledby="navbarDropdown">
		    <!--
  	      <a class="dropdown-item" href="research.html">Research Overview</a>   
  	        <div class="dropdown-divider"></div>          --> 
          	  <a class="dropdown-item" href="evophys.html">Molecular Evolution</a>
          	  <a class="dropdown-item" href="bioinfds.html">Data Science</a>
          	  <a class="dropdown-item" href="biophyschem.html">Biophysical Chemistry</a>
          	  <a class="dropdown-item" href="statphys.html">Statistical Physics</a>
            </div>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="index.html#team">Team</a>
          </li>
	  <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="index.html#fund">Funding</a>
          </li>
	  <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="index.html#collaborators">Collaborators</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="index.html#contact">Contact</a>
          </li>
	  <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="publications.html">Publications</a>
          </li>
	  <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="facilities.html">Facilities</a>
          </li>
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="toolsmethods.html">Tools & Methods</a>
          </li>
	  <!--
          <li class="nav-item">
            <a class="nav-link js-scroll-trigger" href="tutorials.html">Tutorials</a>
          </li> -->
        </ul>
      </div>
    </div>
  </nav>

<!-- Evolutionary Physics -->
  <section class="page-section" id="Molecular Evolution">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h9 class="section-heading text-uppercase"><br>Molecular Evolution</h9>
        </div>
      </div>
    </div>
  </section>
 
  <!-- coevolving structural dynamics -->
  <section class="page-section" id="coevolving structural dynamics">
    <div class="container">
      <div class="row">
        <div class="col-lg-12">
          <h7 class="section-heading text-uppercase" style="text-align: center;">
		  <br>ARE NETWORKS OF COEVOLVING RESIDUES INFORMATIVE OF PROTEINS’ STRUCTURAL DYNAMICS? <br></h7>
            <img class="img-fluid" src="img/research2.png" alt="" style="max-width:100%; max-height:100%;">
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 3vmin;"> <br>
		The intimate relation between proteins' function and structural dynamics suggests that specific networks of 
		residue-residue interactions might be more conserved than the primary structure. The functional mechanics of proteins is 
		exquisitely sensitive to the topology of this network, thus a change in one position of the chain ought to be associated 
		by a compensatory one in a different position. These constraints have arguably left a distinct imprint on the distribution
		of sequences encoding for each specific family of proteins. It is then of crucial relevance to characterize patterns and 
		regularities in large multiple sequence alignments reflecting these specific evolutionary 'design principles'. In 
		particular, to identify networks of co-evolving residues, an extremely fruitful approach is based on a probabilistic 
		model based on a set of pairwise interactions. The least constrained functional form that reproduces single- and
		two-site joint frequencies is that of a 21-states Potts model. We have used these approaches to identify the sequence 
		determinants of voltage sensitivity in the family of Voltage Gated Ion Channels and to glean insight into the activation 
		mechanism of TRP channels. We are actively developing novel methodological approaches to further explore the link between 
		coevolution and structural dynamics. <br>
	  </p>
	  <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 3vmin; font-style: italic">
	      <br>E. Palovcak, L. Delemotte, M. Klein, and V. Carnevale. Comparative sequence analysis suggests a conserved gating 
		  mechanism for TRP channels. JOURNAL OF GENERAL PHYSIOLOGY, 2015, DOI:10.1085/jgp.201411329.<br>
	      <br>E. Palovcak, L. Delemotte, M.L. Klein, V. Carnevale Evolutionary imprint of activation: The design principles of 
		  VSDs THE JOURNAL OF GENERAL PHYSIOLOGY 143 (2), 145-156.<br>
	      <br>E. Palovcak, L. Delemotte, M. L. Klein, and V. Carnevale, Genomics-aided structural modeling of an antiparallel 
		  homodimeric fluoride channelin BIOPHYSICAL JOURNAL, vol. 106, p. 148a, 2014.<br>
	      <br>V. Carnevale, E. Palovcak, L. Delemotte, and M. Klein, Networks of coevolving residues in voltage sensor domains.in 
		  BIOPHYSICAL JOURNAL, vol. 106, p. 743a, 2014.<br>
	      <br>E. Palovcak and V. Carnevale, Correlating residue coevolution and function in a conserved voltage-sensing domain. in 
		  BIOPHYSICAL JOURNAL, vol. 104, p. 277, 2013.<br>  
	  </p>
        </div>
      </div>
    </div>
  </section>

  <!-- coevolving structural dynamics -->
  <section class="page-section" id="coevolving structural dynamics">
    <div class="container">
      <div class="row">
        <div class="col-lg-12">
          <h7 class="section-heading text-uppercase" style="text-align: center;">FUNCTIONAL MECHANICS OF ION CHANNELS FROM THE 6-TM 
		  FAMILY<br></h7>
            <img class="img-fluid" src="img/research4.png" alt="" style="max-width:100%; max-height:100%;">
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 3vmin;"> <br>
		One of the most interesting case studies to investigate the connections between sequence, structural dynamics and 
		function is the six-transmembrane-helix (6-TM) family of ion channels. Ion channels are ubiquitous proteins--one of 
		Nature's exquisite nano-scale molecular machines--that reside in membranes of excitable cells. Their role is to convert 
		chemical and electrical stimuli into ionic currents. Conformational changes in the part of the protein responsible for 
		sensing a stimulus (transducer domain) trigger the gate (effector domain), which can open and close, thereby 
		controlling the flux of ions across the cell membrane. Details of this so-called allosteric communication are of broad 
		relevance to physiology: several drug molecules target ion channels, including neurotoxins and anesthetics, and work by 
		interfering with the coupling between the transducer and effector domains. The 6-TM family, in particular, is 
		characterized by a common structural template inherited from a single ancestor gene that gave rise, through 
		differentiation, to a myriad of functions, from reporting noxious environmental conditions, to shaping the neuronal 
		action potential, to syncing the beating of the heart.
		<br>
		Voltage gated ion channels (VGICs), the proteins responsible for the propagation of electrical signal in neuronal axons,
		belong to the 6-TM family. Despite the fact that these channels have been investigated for several decades, their 
		function relies on molecular mechanisms that are partially unknown. In this context, our interests are focused on the 
		energetics of the activation process of these channels and on the nature of the mechanical coupling between the voltage 
		sensor domain and the pore domain. Besides the voltage 
		gated ion channels, the 6-TM family comprises other physiologically relevant channels, like, for instance, the transient receptor potential (TRP) channels. TRPs play a central role in transducing diverse sensory stimuli in eukaryotes. All known TRP channels act as polymodal cellular sensors and form tetrameric assemblies similar to voltage-gated ion channels.
		<br>
		The fact that TRPs and VGICs are structurally similar, yet functionally distinct raises immediately a question: Is the 
		molecular mechanism underlying polymodal gating in TRP channels different from that underpinning voltage sensitivity 
		in voltage gated ion channels? Intriguingly, comparative sequence analysis on large, comprehensive ensembles of TRP 
		and VGIC channel sequences highlights sequence features that are specific to TRP channels and suggests a model of TRP 
		channel gating that differs substantially from the one mediating voltage sensitivity in VGICs. Most of our recent 
		research efforts are directed toward understanding, via molecular dynamics simulations, the molecular details of this 
		activation mechanism, characterizing the equilibrium and kinetic properties of this transition and, most importantly, 
		shedding light on the role on activation of factors like lipid composition, temperature and pH.<br>
	  </p>
	  <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 3vmin; font-style: italic">
	      <br>E. Palovcak, L. Delemotte, M. Klein, and V. Carnevale. Comparative sequence analysis suggests a conserved gating 
		  mechanism for TRP channels. JOURNAL OF GENERAL PHYSIOLOGY, 2015, DOI:10.1085/jgp.201411329.<br>
	      <br>L. Delemotte, M. A. Kasimova, M. L. Klein, M. Tarek, V. Carnevale Free Energy Landscape of Ion-Channel 
		  Voltage-Sensor-Domain Activation PROC. NATL. ACAD. SCI. USA, doi: 10.1073/pnas.1416959112.<br>
	      <br>C. Amaral, V. Carnevale, M. L. Klein, W. Treptow Exploring conformational states of the bacterial voltage-gated 
		  sodium channel NavAb via molecular dynamics simulations PROC. NATL. ACAD SCI. USA 109(52), 21336-21341 (2012).<br>
	      <br>A. Barber, V. Carnevale, S.G. Raju, C. Amaral, Werner Treptow, M. L. Klein Hinge-bending motions in the pore domain 
		  of a bacterial voltage-gated sodium channel.BIOCHIMICA ET BIOPHYSICA ACTA (BBA) - BIOMEMBRANES, 1818(9), 2120-2125 
		  (2012).<br>  
	  </p>
        </div>
      </div>
    </div>
  </section>

  <!-- Footer -->
  <footer class="footer">
    <div class="container">
      <div class="row align-items-center">
        <div class="col-md-4">
          <span class="copyright">Copyright &copy; Vincenzo Carnevale</span>
        </div>
        <div class="col-md-4">
          <ul class="list-inline social-buttons">
            <li class="list-inline-item">
              <a href = "https://github.com/carnevale-lab">
                <i class = "fab fa-github"></i>
              </a>
            </li>
          </ul>
        </div>
        <div class="col-md-4">
        </div>
      </div>
    </div>
  </footer>

 
  </div>

  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

  <!-- Contact form JavaScript -->
  <script src="js/jqBootstrapValidation.js"></script>
  <script src="js/contact_me.js"></script>

  <!-- Custom scripts for this template -->
  <script src="js/agency.min.js"></script>

</body>

</html>
