<!DOCTYPE html>
<html lang="en">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="">
    
    <!--Jquery used to link the navBar to all other pages  -->
    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>

  <!-- Bootstrap core CSS -->
  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom fonts for this template -->
  <script src="https://kit.fontawesome.com/e51c75a363.js" crossorigin="anonymous"></script>
  <!--<link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css"> -->
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
  <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
  <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
  
  <!-- Custom styles for this template -->
  <link href="css/agency.css" rel="stylesheet">

  <style>
  table {
    font-family: arial, sans-serif;
    border-collapse: collapse;
    width: 45%;
    margin: 10px;
  }
  
  td, th {
    border: 1px solid #dddddd;
    text-align: left;
    padding: 8px;
  }
  
  tr:nth-child(even) {
    background-color: #dddddd;
  </style>

<!-- Navbar Script, using Jquery to load from nav.html -->
<div id="nav-placeholder">

</div>

<script>
$(function(){
  $("#nav-placeholder").load("nav.html");
});
</script>    
    
  
</head>
  

 <!-- Molecular Evolution & Machine Learning -->
  <section class="page-section" id="topicheader">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h9 class="section-heading text-uppercase"><br>Molecular Evolution & Machine Learning</h9>
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 3vmin;"><br><!-- can add forward text here --> </p>
        </div>
      </div>
    </div>
  </section>
 
  <!-- Patterns of coevolving, PNAS 2017 -->
  <section class="page-section" id="molecevol">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h7 class="section-heading" style="font-size: 4vmin">Patterns of coevolving amino acids unveil structural and dynamical domains</h7>
          <h8 class="section-subheading" style="font-style: italic"><br>Daniele Granata, Luca Ponzoni, Cristian Micheletti, and Vincenzo Carnevale<br>PNAS, 114(50), E10612-E10621, 2017<br><br></h8>
            <img class="right" src="img/spectrus-evo.jpg" alt="" style="max-width:60%; max-height:50%; float:right; padding: 0 0 0 12px"/>
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 2vmin;">
		Patterns of interacting amino acids are so preserved within protein families that the sole analysis of evolutionary comutations can identify pairs of contacting residues. It is also known that evolution conserves functional dynamics, i.e., the concerted motion or displacement of large protein regions or domains. Is it, therefore, possible to use a pure sequence-based analysis to identify these dynamical domains? To address this question, we introduce here a general coevolutionary coupling analysis strategy and apply it to a curated sequence database of hundreds of protein families. For most families, the sequence-based method partitions amino acids into a few clusters. When viewed in the context of the native structure, these clusters have the signature characteristics of viable protein domains: They are spatially separated but individually compact. They have a direct functional bearing too, as shown for various reference cases. We conclude that even large-scale structural and functionally related properties can be recovered from inference methods applied to evolutionary-related sequences.<br></p>
        </div>
      </div>
    </div>
  </section>

  <!-- Intrinsic dimension, Sci Rep 2016 -->
  <section class="page-section" id="molecevol">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h7 class="section-heading" style="font-size: 4vmin">Accurate estimation of the intrinsic dimension using graph distances</h7>
          <h8 class="section-subheading" style="font-style: italic"><br>Daniele Granata and Vincenzo Carnevale<br>Scientific Reports, 6, 31377, 2016<br><br></h8>
            <img class="right" src="img/ID2.jpg" alt="" style="max-width:60%; max-height:50%; float:right; padding: 0 0 0 12px"/>
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 2vmin;">
		The collective behavior of a large number of degrees of freedom can be often described by a handful of variables. This observation justifies the use of dimensionality reduction approaches to model complex systems and motivates the search for a small set of relevant “collective” variables. Here, we analyze this issue by focusing on the optimal number of variable needed to capture the salient features of a generic dataset and develop a novel estimator for the intrinsic dimension (ID). By approximating geodesics with minimum distance paths on a graph, we analyze the distribution of pairwise distances around the maximum and exploit its dependency on the dimensionality to obtain an ID estimate. We show that the estimator does not depend on the shape of the intrinsic manifold and is highly accurate, even for exceedingly small sample sizes. We apply the method to several relevant datasets from image recognition databases and protein multiple sequence alignments and discuss possible interpretations for the estimated dimension in light of the correlations among input variables and of the information content of the dataset.<br></p>
        </div>
      </div>
    </div>
  </section>
  
  <!-- Generative Capacity, arXiv 2020 -->
  <section class="page-section" id="molecevol">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h7 class="section-heading" style="font-size: 4vmin">Generative capacity of probabilistic protein sequence models</h7>
          <h8 class="section-subheading" style="font-style: italic"><br>Francisco McGee, Quentin Novinger, Ronald Levy, Vincenzo Carnevale, and Allan Haldane<br>arXiv preprint, arXiv:2012.02296, 2020<br><br></h8>
            <img class="right" src="img/generative.png" alt="" style="max-width:60%; max-height:50%; float:right; padding: 0 0 0 12px"/>
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 2vmin;">
		Potts models and variational autoencoders (VAEs) have recently gained popularity as generative protein sequence models (GPSMs) to explore fitness landscapes and predict the effect of mutations. Despite encouraging results, quantitative characterization and comparison of GPSM-generated probability distributions is still lacking. It is currently unclear whether GPSMs can faithfully reproduce the complex multi-residue mutation patterns observed in natural sequences arising due to epistasis. We develop a set of sequence statistics to assess the "generative capacity" of three GPSMs of recent interest: the pairwise Potts Hamiltonian, the VAE, and the site-independent model, using natural and synthetic datasets. We show that the generative capacity of the Potts Hamiltonian model is the largest, in that the higher order mutational statistics generated by the model agree with those observed for natural sequences. In contrast, we show that the VAE's generative capacity lies between the pairwise Potts and site-independent models. Importantly, our work measures GPSM generative capacity in terms of higher-order sequence covariation statistics which we have developed, and provides a new framework for evaluating and interpreting GPSM accuracy that emphasizes the role of epistasis.<br></p>
        </div>
      </div>
    </div>
  </section>
  
  <!-- Comparative Sequence Analysis, JGP 2015 -->
  <section class="page-section" id="molecevol">
    <div class="container">
      <div class="row">
        <div class="col-lg-12 text-center">
          <h7 class="section-heading" style="font-size: 4vmin">Comparative sequence analysis suggests a conserved gating mechanism for TRP channels</h7>
          <h8 class="section-subheading" style="font-style: italic"><br>Eugene Palovcak, Lucie Delemotte, Michael Klein, and Vincenzo Carnevale<br>The Journal of General Physiology, 146(1), 37-50, 2015<br><br></h8>
            <img class="right" src="img/research4.png" alt="" style="max-width:60%; max-height:50%; float:right; padding: 0 0 0 12px"/>
          <p style="text-align: justify !important; line-height: 1.5; font-weight: 250; font-size: 2vmin;">
		The transient receptor potential (TRP) channel superfamily plays a central role in transducing diverse sensory
		stimuli in eukaryotes. Although dissimilar in sequence and domain organization, all known TRP channels act as
		polymodal cellular sensors and form tetrameric assemblies similar to those of their distant relatives, the voltage-
		gated potassium (Kv) channels. Here, we investigated the related questions of whether the allosteric mechanism
		underlying polymodal gating is common to all TRP channels, and how this mechanism differs from that underpinning 
		Kv channel voltage sensitivity. To provide insight into these questions, we performed comparative sequence
		analysis on large, comprehensive ensembles of TRP and Kv channel sequences, contextualizing the patterns of
		conservation and correlation observed in the TRP channel sequences in light of the well-studied Kv channels. We
		report sequence features that are specific to TRP channels and, based on insight from recent TRPV1 structures, we
		suggest a model of TRP channel gating that differs substantially from the one mediating voltage sensitivity in Kv
		channels. The common mechanism underlying polymodal gating involves the displacement of a defect in the
		H-bond network of S6 that changes the orientation of the pore-lining residues at the hydrophobic gate. <br></p>
        </div>
      </div>
    </div>
  </section>




  <!-- Footer -->
  <footer class="footer">
    <div class="container">
      <div class="row align-items-center">
        <div class="col-md-4">
          <span class="copyright">Copyright &copy; Vincenzo Carnevale</span>
        </div>
        <div class="col-md-4">
          <ul class="list-inline social-buttons">
            <li class="list-inline-item">
              <a href = "https://github.com/carnevale-lab">
                <i class = "fab fa-github"></i>
              </a>
            </li>
          </ul>
        </div>
        <div class="col-md-4">
        </div>
      </div>
    </div>
  </footer>

 
  </div>

  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

  <!-- Contact form JavaScript -->
  <script src="js/jqBootstrapValidation.js"></script>
  <script src="js/contact_me.js"></script>

  <!-- Custom scripts for this template -->
  <script src="js/agency.min.js"></script>

</body>

</html>
